modules/core/src/main/scala/Json.scala

```diff
@@ -0,0 +1,134 @@
+package scalaz
+
+package schema
+
+import scalaz.Scalaz._
+import scalaz.schema.JsonSchema._
+
+object Json {
+  type JSON = String
+
+  val module = new SchemaModule {
+    type Prim[A]       = JsonSchema.Prim[A]
+    type ProductTermId = String
+    type SumTermId     = String
+  }
+
+  trait ToJson[S[_]] {
+    def serializer: S ~> (? => JSON)
+  }
+
+  implicit val toJson: ToJson[module.Prim] = new ToJson[module.Prim] {
+    override def serializer: module.Prim ~> (? => JSON) = new (module.Prim ~> (? => JSON)) {
+      override def apply[A](fa: module.Prim[A]): A => JSON = fa match {
+        case JsonString => a => s""""$a""""
+        case JsonBool   => a => if (a) "true" else "false"
+        case JsonNull   => _ => null
```

Comment:
This confused me totally. I expected that serialization of object `Person(null, Some(User(true))` should result in `NullPointerException` but it seems that `.mkString` uses `String.valueOf` function that converts `null` to `"null"` string. 
I would prefer that this part serializes value directly to string `"null"`. 
Reason is that for me when function returns `null` value it means that it is not defined on given input and that result type should be `Option[A]`.

Reply:
this was actually just me forgetting to add `"` :D I'll fix that and make it more clear